test/integration/ovs/ofctrl_test.go

```diff
@@ -1324,3 +1325,12 @@ func getCookieIDMask() (uint64, uint64) {
 	cookieMask := uint64(^uint16(0))<<48 | uint64(^uint8(0))<<40
 	return cookieID, cookieMask
 }
+
+func getFlowModificationMessages(flows []binding.Flow, operation binding.OFOperation) []*openflow15.FlowMod {
+	messages := make([]*openflow15.FlowMod, 0, len(flows))
+	for _, f := range flows {
+		msg, _ := f.GetBundleMessages(operation)
```

Comment:
Is it safe to ignore the error?

Reply:
This is the test code, we the flow is prepared as expected. So it is not supposed to return error.